### PR TITLE
Archived Asset is presented in the list

### PR DESCRIPTION
**Pull request by yonetmen**

### CODE REVIEW

**Comment by yonetmen** [comment]
test me

**Review by peerik — APPROVED** [review]

**Comment by yonetmen** [comment]
test me

**Comment by yonetmen** [comment]
test me

**Comment by yonetmen** [comment]
We will wait for next weeks meeting (25:e mars)  before we can merge this branch.

**Pull request closed without merging**
